less/components/page-end-cta.less

```diff
@@ -3,26 +3,4 @@
     .callout-heading;
   }
   text-align: center;
-  form {
-    text-align: left;
-    fieldset:last-child {
-      margin-bottom: 1.5rem;
-      @media (min-width: @screen-sm-min) {
-        margin-bottom: 0;
-      }
-    }
-    input {
-      padding: 1rem 1.5rem;
-      border: .1rem solid @grey;
-    }
-    input[type="email"] {
-      width: 100%;
-    }
-    input[type="checkbox"] + div {
-      text-align: left;
-    }
-    input[type="submit"] {
-      margin-top: 0;
-    }
-  }
```

Comment:
I realized the signup form on Events page is the **only** form on the site that's embedded within `.page-end-cta`.  So I think it makes sense to move these rules to `less/pages/events.less`.  That way if we ever have to have another form in `<PageEndCTA>`, we won't have to override stylings.